pkgs/top-level/all-packages.nix

```diff
@@ -4016,6 +4016,8 @@ in
 
   ugarit-manifest-maker = callPackage ../tools/backup/ugarit-manifest-maker { };
 
+  unar = callPackage ../tools/archivers/unar { };
```

Comment:
Can you overwrite the stdenv to "clangStdenv" like is done in gnustep? That way we can get rid of the "clang" build dependency.